src/components/HookCounter/HookCounter.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { useState } from 'react'
+
+function HookCounter() {
```

Comment:
try to use arrow functions and follow ES6 syntax

Reply:
I made the respective changes wherever needed.